src/pages/api/patch-notes.js

```diff
@@ -15,17 +13,11 @@ export default async function handler(request, response) {
   try {
     const [{ slug }] = await getReleases({ isPreview })
 
-    trackAsync(request, 'pageview', path)
```

Comment:
This is now built in, since we use the full Vercel stack (Next.js as a framework, Vercel as a hosting provider and Vercel Analytics for tracking).